src/libs/actions/Report.js

```diff
@@ -1132,6 +1075,49 @@ function openReport(reportID) {
         });
 }
 
+/**
+ * Gets the actions from the oldest unread action.
+ *
+ * @param {Number} reportID
+ * @param {Number} oldestActionSequenceNumber
+ */
+function readOldestAction(reportID, oldestActionSequenceNumber) {
+    API.read('ReadOldestAction',
+        {
+            reportID,
+            reportActionsOffset: oldestActionSequenceNumber,
+        },
+        {
+            optimisticData: [{
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.IS_LOADING_REPORT_ACTIONS}${reportID}`,
+                value: {isLoading: true},
+            }],
+            successData: [
+                {
+                    onyxMethod: CONST.ONYX.METHOD.MERGE,
+                    key: `${ONYXKEYS.COLLECTION.IS_LOADING_REPORT_ACTIONS}${reportID}`,
+                    value: {
+                        isLoading: false,
+                        error: '',
```

Comment:
I'm not sure if we are setting errors for this or where it would happen 🤔  

I think if we can't read the actions then the user will just see the older actions until they can update.

Reply:
how does the user know they can try back later?
I don't have anything against not having the error though

Reply:
it makes me wonder about the user seeing a spinning loader and then nothing changing, and makes me think if we should do a request first with no spinner to see if there is any change (if the sequence number is different from the backend one) and if there is then we show the spinner and load the actions.

Reply:
>how does the user know they can try back later?

When they regain connection they will retry automatically.

>makes me think if we should do a request first with no spinner to see if there is any change

This is somewhat solved by the open PR I linked you as we will only show a spinner when there are no chat messages stored locally.

Regardless, I think we don't have to worry about it here as we didn't scope this out and don't really expect this to fail as long as a user is online. It does bring up a good question about what would happen if we are offline. But I don't think we should block on that conversation and should keep going with this PR for now.